docs/worklow-configuration-how-to.md

```diff
@@ -7,7 +7,26 @@ How to configure `workflows.yml` workflow configurations for specifying buckets.
 
 ### Point to buckets in the configuration
 
-Buckets specified in `app/config.yml` will become part of the `meta` object of the Cumulus message and can be accessed in your workflow configuration.
+Buckets specified in `app/config.yml` will become part of the `meta` object of the Cumulus message and can be accessed in your workflow configuration. Consider this `app/config.yml` for the following examples:
+
+```
+buckets:
```

Comment:
I think we should say something like bucket types should be internal, public, private, or protected. Maybe a description of what that means - internal is for system use, public would be publicly available, protected behind ED login, and private for... private info.